src/modules/commander/PreflightCheck.cpp

```diff
@@ -427,11 +413,11 @@ static bool airspeedCheck(orb_advert_t *mavlink_log_pub, vehicle_status_s &statu
 	}
 
 	/**
-	 * Check if differential pressure is off by more than 15Pa which equals ~5m/s when measuring no airspeed.
+	 * Check if airspeed is higher than 4m/s (accepted max) while the vehicle is landed / not flying
 	 * Negative and positive offsets are considered. Do not check anymore while arming because pitot cover
 	 * might have been removed.
 	 */
-	if (fabsf(differential_pressure.differential_pressure_filtered_pa) > 15.0f && !prearm) {
+	if (fabsf(airspeed.indicated_airspeed_m_s) > 4.0f && !prearm) {
 		if (report_fail) {
 			mavlink_log_critical(mavlink_log_pub, "Preflight Fail: check Airspeed Cal or Pitot");
```

Comment:
The short-hand "Cal" is confusing to most users.  Also, this warning as currently written isn't consistent with the style of the other preflight warnings.  What do you think about making this Preflight Fail: warning more specific to the actual failure?  Here are some suggested alternatives:

"Preflight Fail: airspeed too high."
"Preflight Fail: airspeed too large."
"Preflight Fail: airspeed reading too high."
"Preflight Fail: airspeed reading too large."
"Preflight Fail: airspeed out of range."
"Preflight Fail: airspeed reading out of range."